docs/spec.md

```diff
@@ -116,6 +116,69 @@ types, for example, to include layouts
 ([#629](https://github.com/openxla/stablehlo/issues/629)) and sparsity
 ([#1078](https://github.com/openxla/stablehlo/issues/1078)).
 
+```ebnf
+QuantizedTensorType ::= 'tensor' '<' TensorShape QuantizedElementType '>'
+QuantizedElementType ::= '!quant.uniform' '<'
+                  QuantizationStorageType
+                  ['<' QuantizationStorageMin ':' QuantizationStorageMax '>']
+                  ':' QuantizationExpressedType
+                  [':' QuantizationDimension]
+                  ',' QuantizationParameters '>'
+QuantizationStorageType ::= IntegerType
+QuantizationStorageMin ::= IntegerConstant
+QuantizationStorageMax ::= IntegerConstant
+QuantizationExpressedType ::= FloatType
+QuantizationDimension ::= IntegerConstant
+QuantizationParameters ::= QuantizationParameter
+                         | '{' QuantizationParameter {',' QuantizationParameter} '}'
+QuantizationParameter ::= QuantizationScale ':' QuantizationZeroPoint
+QuantizationScale ::= FloatConstant
```

Comment:
Can we open the possibility to have unknown values for scale and zero point for quantized training, which requires the training programs to provide scale and zero point values on the fly?

Reply:
I think that's an interesting idea, and #1149 is highlighting this as an open issue as well. 

However, given that we don't have a precedent for this functionality (quant.uniform doesn't support this yet, and therefore StableHLO dialect doesn't support this either), I think that this functionality would need to be introduced in a dedicated RFC once we define the baseline StableHLO spec.

This is similar to how we've forked StableHLO from MHLO and specced baseline HLO semantics, and now are starting to make changes to it along the lines of #1143, #1308, #1342, etc.